### ISSUE TITLE
Remove deprecated css var `--target-size`

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1237]
see: https://github.com/dequelabs/cauldron/blob/033b4e816c59e800b883b09f0b7764fd0728e7b1/packages/styles/variables.css#L142

`target-size` should be replaced by `target-size-minium` and `target-size-enhanced` to target WCAG 2.2 AA and AAA respectively.
